src/Components/Publishing/Series/SeriesAbout.tsx

```diff
@@ -6,11 +6,13 @@ import { media } from "../../Helpers"
 import { Fonts } from "../Fonts"
 import { PartnerBlock, PartnerBlockContainer } from "../Partner/PartnerBlock"
 import { Text } from "../Sections/Text"
+import { ArticleData } from "../Typings"
 
 interface Props {
-  article?: any
+  article?: ArticleData
   color?: string
   editDescription?: any
+  editSubTitle?: any
```

Comment:
Would this be anything other than a string? I was just a little curious about the any typing?

Reply:
The edit props are used for passing functions -- in this case it would be a Draft.js component from positron.

Reply:
gotcha 👍